cloud/deployment/deployment.go

```diff
@@ -91,102 +105,119 @@ func CanCiCdDeploy(bearerToken string) bool {
 	c, err := config.GetCurrentContext()
 	if err != nil {
 		return err
 	}
 	tab := newTableOut()
+
 	if all {
 		ws = ""
 		tab = newTableOutAll()
 	}
-	deployments, err := GetDeployments(ws, c.Organization, client)
+	deployments, err := CoreGetDeployments(ws, c.Organization, platformCoreClient)
 	if err != nil {
-		return errors.Wrap(err, astro.AstronomerConnectionErrMsg)
+		return err
 	}
 
 	if len(deployments) == 0 {
 		fmt.Printf("No Deployments found in workspace %s\n", ansi.Bold(ws))
 		return nil
 	}
 
-	sort.Slice(deployments, func(i, j int) bool { return deployments[i].Label > deployments[j].Label })
+	sort.Slice(deployments, func(i, j int) bool { return deployments[i].Name > deployments[j].Name })
 
 	// Build rows
 	for i := range deployments {
 		d := deployments[i]
-		clusterName := d.Cluster.Name
-		runtimeVersionText := d.RuntimeRelease.Version + " (based on Airflow " + d.RuntimeRelease.AirflowVersion + ")"
-		releaseName := d.ReleaseName
-		if organization.IsOrgHosted() {
-			if IsDeploymentHosted(d.Type) {
-				clusterName = d.Cluster.Region
-			}
-			releaseName = notApplicable
+		// change to cluster name
+		var clusterName string
+		if !IsDeploymentStandard(*d.Type) {
+			clusterName = *d.ClusterName
+		} else {
+			clusterName = notApplicable
+		}
+		runtimeVersionText := " (based on Airflow " + d.RuntimeVersion + ")"
+		releaseName := d.Namespace
+		// change to workspace name
+		workspaceID := d.WorkspaceId
+		region := notApplicable
+		cloudProvider := notApplicable
+		if IsDeploymentStandard(*d.Type) || IsDeploymentDedicated(*d.Type) {
+			region = *d.Region
+			cloudProvider = *d.CloudProvider
 		}
-
 		if all {
-			tab.AddRow([]string{d.Label, d.Workspace.Label, releaseName, clusterName, d.ID, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.APIKeyOnlyDeployments)}, false)
+			tab.AddRow([]string{d.Name, workspaceID, releaseName, clusterName, cloudProvider, region, d.Id, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.IsCicdEnforced), string(*d.Type)}, false)
 		} else {
-			tab.AddRow([]string{d.Label, releaseName, clusterName, d.ID, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.APIKeyOnlyDeployments)}, false)
+			tab.AddRow([]string{d.Name, releaseName, clusterName, cloudProvider, region, d.Id, runtimeVersionText, strconv.FormatBool(d.DagDeployEnabled), strconv.FormatBool(d.IsCicdEnforced), string(*d.Type)}, false)
 		}
 	}
-
-	return tab.Print(out)
+	tab.Print(out)
+	return nil
 }
 
-func Logs(deploymentID, ws, deploymentName string, warnLogs, errorLogs, infoLogs bool, logCount int, client astro.Client) error {
-	logLevels := []string{}
-
+func Logs(deploymentID, ws, deploymentName string, warnLogs, errorLogs, infoLogs bool, logCount int, platformCoreClient astroplatformcore.CoreClient, coreClient astrocore.CoreClient) error {
+	var logLevel string
```

Comment:
btw the reason the logLevels are not in core api is because of this PR https://github.com/astronomer/astro/pull/11949
where it states trying to filter logs by log level doesnt make much sense

Reply:
Just trying to not remove flags but we can if not necessary